docs/aws-sdk-for-iOS-with-minio.md

```diff
@@ -0,0 +1,58 @@
+# How to use AWS SDK for iOS(swift-3) with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+In this recipe we will learn how to use `AWSS3` for iOS with Minio server. `AWSS3` is the official AWS S3 SDK for the swift/objective-c programming language.
+
+## 1. Prerequisites
+
+Install Minio Server from [here](http://docs.minio.io/docs/minio).
```

Comment:
The link doesn't work, correct link is : https://docs.minio.io/docs/minio-quickstart-guide